test/Oscoin/Test/Crypto/Blockchain/Block/Generators.hs

```diff
@@ -61,6 +67,36 @@ genStandaloneBlock = do
                }
     pure $ mkBlock header blockDataBeneficiary blockDataTxs
 
+someBeneficiary :: Crypto.HasHashing c => Beneficiary c
+someBeneficiary = Crypto.zeroShortHash
```

Comment:
Bikeshedding on my bikeshed .. here the purpose is clearly to distinguish between arbitrary beneficiaries vs a "stock", fixed one .. perhaps `fixedBeneficiary` or `stockBeneficiary` is better fitting?

Reply:
How about `beneficiaryFixture`?

Reply:
I guess it boils down to personal preferences -- for some reason I prefer an `adjective + noun` schema here, but I won't antagonise if you pick `beneficiaryFixture` 😉